devops/build.sh

```diff
@@ -6,3 +6,6 @@ MAIN_FOLDER="${SCRIPT_FOLDER}/.."
 
 echo "Running build on folder ${MAIN_FOLDER}"
 ( cd "${MAIN_FOLDER}" && docker build -t lms . )
+
+echo "Compiling Flask Babel"
```

Comment:
When coming to think about it, we should add `i18n.sh`, and run it in `upgrade.py`.